<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>UserProfileView</class>
    <widget class="QWidget" name="UserProfileView">
        <property name="windowTitle">
            <string>个人信息维护</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QHBoxLayout" name="horizontalLayoutUsername">
                    <item>
                        <widget class="QLabel" name="labelUsernameTitle">
                            <property name="text">
                                <string>用户名：</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="usernameLabel"/>
                    </item>
                    <item>
                        <widget class="QPushButton" name="updateUsernameButton">
                            <property name="text">
                                <string>修改用户名</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayoutPhone">
                    <item>
                        <widget class="QLabel" name="labelPhoneTitle">
                            <property name="text">
                                <string>手机号：</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="phoneLabel"/>
                    </item>
                    <item>
                        <widget class="QPushButton" name="updatePhoneButton">
                            <property name="text">
                                <string>修改手机号</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayoutCompany">
                    <item>
                        <widget class="QLabel" name="labelCompanyTitle">
                            <property name="text">
                                <string>所属公司：</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="companyLabel"/>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QPushButton" name="updatePasswordButton">
                    <property name="text">
                        <string>修改密码</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui> 